annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -322,15 +326,15 @@ protected Agent buildAgent(String id) {
 	    JsonLd annotationLd = new AnnotationLdSerializer(annotation);
 	    String jsonLd = annotationLd.toString(4);
 
-	    int etag;
+	    String eTag;
 	    if (annotation.getLastUpdate() != null)
```

Comment:
please remove the use of lastUpdated, this was replaced now by generated

Reply:
replaced by getGenerated